src/App.js

```diff
@@ -1,11 +1,10 @@
-
-import { useRef, useState, useEffect } from "react";
+import React from "react";
```

Comment:
I think there was a misunderstanding about my comment about imports. This won't work. You don't need to import `React`, but you do need to keep `{ useRef, useState, useEffect }`.

Reply:
Yes, there was some misunderstanding and since a hour it was bugging me how hooks are gonna work, I'll correct it ASAP.